roles/kubernetes/control-plane/tasks/0010-etcd-link.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create etcd cert directory
+  ansible.builtin.file:
+    path: "{{ etcd_cert_dir }}"
+    state: directory
+    mode: '0750'
+  when:
+    - inventory_hostname  in groups['kube_control_plane']
+
+- name: Generate symlink to etcd certs
+  ansible.builtin.file:
+    src: "{{item.src}}"
+    dest: "{{item.dest}}"
+    state: link
+    force: true
+  loop:
+    - src: "{{etcd_cert_paths.client.ca}}"
+      dest: "{{ etcd_cert_dir }}/{{kube_etcd_cacert_file}}"
+    - src: "{{etcd_cert_paths.client.cert}}"
+      dest: "{{ etcd_cert_dir }}/{{kube_etcd_cert_file}}"
+    - src: "{{etcd_cert_paths.client.key}}"
+      dest: "{{ etcd_cert_dir }}/{{kube_etcd_key_file}}"
```

Comment:
I'd factor what we can out of the loop item, but that's probably preference.
```
src: {{ etcd_cert_paths.client[item.src] }}
dst: {{ etcd_cert_dirs }}/{{ item.dst }}
```
With a loop like
```
- src: ca
  dst: "{{ kube_etcd_cacert_file }}"
```

Reply:
Also, can you fix the jinja spacing ("{{ var }}") ? That's a bit easier to read with spacing around the var.